src/sql/workbench/contrib/assessment/test/common/vsServicesStubs.ts

```diff
@@ -0,0 +1,205 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { IEnvironmentService, IExtensionHostDebugParams } from 'vs/platform/environment/common/environment';
+import { URI } from 'vs/base/common/uri';
+import * as files from 'vs/platform/files/common/files';
+import { IDisposable } from 'vs/base/common/lifecycle';
+import { VSBuffer, VSBufferReadable, VSBufferReadableStream } from 'vs/base/common/buffer';
+import { Event } from 'vs/base/common/event';
+import { IDialogService, IConfirmation, IDialogOptions, IConfirmationResult, IShowResult } from 'vs/platform/dialogs/common/dialogs';
+import Severity from 'vs/base/common/severity';
+import { IClipboardService } from 'vs/platform/clipboard/common/clipboardService';
+import {
+	INotificationService,
+	INotification,
+	INotificationHandle,
+	IPromptOptions,
+	IStatusMessageOptions,
+	NotificationsFilter,
+	NotificationMessage,
+	IPromptChoice
+} from 'vs/platform/notification/common/notification';
+
+
+export class EnvironmentServiceStub implements IEnvironmentService {
+	_serviceBrand: undefined;
+	userRoamingDataHome: URI;
+	settingsResource: URI;
+	keybindingsResource: URI;
+	keyboardLayoutResource: URI;
+	argvResource: URI;
+	snippetsHome: URI;
+	backupHome: URI;
+	untitledWorkspacesHome: URI;
+	globalStorageHome: URI;
+	workspaceStorageHome: URI;
+	userDataSyncLogResource: URI;
+	userDataSyncHome: URI;
+	sync: 'on' | 'off';
+	enableSyncByDefault: boolean;
+	debugExtensionHost: IExtensionHostDebugParams;
+	isExtensionDevelopment: boolean;
+	disableExtensions: boolean | string[];
+	extensionDevelopmentLocationURI?: URI[];
+	extensionTestsLocationURI?: URI;
+	extensionEnabledProposedApi?: string[];
+	logExtensionHostCommunication?: boolean;
+	logsPath: string;
+	logLevel?: string;
+	verbose: boolean;
+	isBuilt: boolean;
+	disableTelemetry: boolean;
+	serviceMachineIdResource: URI;
+
+}
+
+export class FileServiceStub implements files.IFileService {
```

Comment:
And https://github.com/microsoft/azuredatastudio/blob/main/src/vs/workbench/test/browser/workbenchTestServices.ts#L723?